chat-service/package.json

```diff
@@ -7,15 +7,18 @@
   "scripts": {
     "dev": "nodemon index.js",
     "start": "node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha --exit"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "chai": "^4.3.6",
```

Comment:
chai and mocha should be npm i chai mocha --save-dev so that it will fall under devDependencies below

Reply:
Fixed